README.md

```diff
@@ -17,7 +17,19 @@
 
 **2.** Install [`Docker`](https://docs.docker.com/engine/installation/).
 
-**3.** Download the pipeline and our minimal test dataset with a single command run in the root of this repository:
+**3.** Build the required Docker images (optional - only needed if you've modified the pipeline or are using a custom fork):
+
+```bash
+make docker-all
+```
+
+Or build individual images:
+```bash
+make docker-physicochemical-props
+make docker-phylo-dist
+```
+
```

Comment:
Do we actually want to include these instructions? I guess maybe this is helpful/useful, especially for these more involved docker images?